src/lib/MediaPreview/MediaPreview.vue

```diff
@@ -108,6 +184,49 @@ export default {
     },
     onIframeFinishedLoading() {
       this.isLoadingIframe = false
+    },
+    async getFileContent(file) {
+      this.isFetchingFile = true
+
+      const url = file?.downloadUrl ?? file?.previewUrl
+
+      if (this.cachedFiles[url]) {
+        this.fileContent = this.cachedFiles[url]
+        this.isFetchingFile = false
+        return this.fileContent
+      }
+
+      try {
+        await fetch(url)
+          .then(async res => {
+            this.fileContent = await res.text()
+          })
+      } catch (error) {
+        this.fileContent = null
+      }
+
+      this.cachedFiles[url] = this.fileContent
+      this.isFetchingFile = false
+      return this.fileContent
+    },
+    sanitizeSVG(svg) {
+      return (
+        !svg || !svg.length
+          ? ''
+          : svg.toString().replace(/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi, '')
```

Comment:
Antes de mais nada, parabéns por essa preocupação com segurança aqui! 👏 

Segundo, essa parte aqui é um pouco preocupante. Ter essa sanitização do SVG aqui é ótimo e necessário, mas ele não cobre todos os casos. Precisamos de algo mais robusto como uma lib dedicada a encontrar todos os possíveis ataques. Se o VAC tiver o `dompurify` disponível como dependência, podemos fazer:
```suggestion
          : DOMPurify.sanitize(svg.toString())
```

Se o dompurify não estiver disponível aqui, precisamos fazer isso no lado do chat.

cc: @arufonsekun @DouglasKosvoski

Reply:
Adicionei o `dompurify`no V.A.C e ajustei a implementação da sanitização, obrigado pelo conselho em usar um `lib` para isso.